omnibus_overrides.rb

```diff
@@ -4,7 +4,6 @@
 # NOTE: You MUST update omnibus-software when adding new versions of
 # software here: bundle exec rake dependencies:update_omnibus_gemfile_lock
 override :rubygems, version: "3.1.4" # pin to what ships in the ruby version
-override :bundler, version: "2.1.4" # pin to what ships in the ruby version
```

Comment:
don't we still pull in the bundler omnibus-software definition from the ruby definition which will then install whatever its default version is?

Reply:
We had to nuke all that junk when ruby integrated it. It causes issues when we need to build stuff for old ruby releases.